### PR TITLE
fixup README.md

### PR DESCRIPTION
**Pull request by JudsenAtFlexgen**
# Description
Hello there, Really nice repo you got here. I'm just tweaking some documentation here. 
I am running zig version `0.14.0-dev.2051+b1361f237`.
Which is ahead of 0.13, but you would also be able to do my method here on 0.13 as well. So I think my pr is valid.
## Documentation update
`std.mem.spanZ` is no longer in the standard. Found an alternate way to run the example. Updating readme for the next guy. 
Just a 2 line change to the README.md.

### CODE REVIEW

**Comment by vrischmann** [comment]
Hi,

thanks for the fix !

Although after some testing I'd rather change it to this and taking advantage of type coercion:
```zig
if (row) |r| {
    const name_ptr: [*:0]const u8 = &r.name;
    std.log.debug("name: {s}, age: {}", .{ std.mem.span(name_ptr), r.age });
}
```

The whole `span((&row.name).ptr)` is a little hard to parse in my opinion.

File: README.md [hunk 1]
```diff
@@ -227,8 +227,8 @@ const row = try stmt.one(
     .{},
     .{ .id = 20 },
 );
-if (row) |row| {
-    std.log.debug("name: {}, age: {}", .{std.mem.spanZ(&row.name), row.age});
+if (row) |r| {
+    std.log.debug("name: {}, age: {}", .{std.mem.span((&row.name).ptr), row.age});
```
Comment (vrischmann) [inline README.md:231]:
```suggestion
if (row) |r| {
    const name_ptr: [*:0]const u8 = &r.name;
    std.log.debug("name: {s}, age: {}", .{ std.mem.span(name_ptr), r.age });
}
```

**Comment by JudsenAtFlexgen** [comment]
> Hi,
> 
> thanks for the fix !
> 
> Although after some testing I'd rather change it to this and taking advantage of type coercion:
> 
> ```zig
> if (row) |r| {
>     const name_ptr: [*:0]const u8 = &r.name;
>     std.log.debug("name: {s}, age: {}", .{ std.mem.span(name_ptr), r.age });
> }
> ```
> 
> The whole `span((&row.name).ptr)` is a little hard to parse in my opinion.

LGTM. 

Thanks!

**Review by vrischmann — APPROVED** [review]
